app.comaps.desktop.yml

```diff
@@ -0,0 +1,46 @@
+app-id: app.comaps.desktop
+runtime: org.kde.Platform
+runtime-version: '6.9'
+sdk: org.kde.Sdk
+command: CoMaps
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --system-talk-name=org.freedesktop.GeoClue2.*
+modules:
+  - name: optipng
+    build-options:
+      config-opts:
+       - --with-system-libs
+    sources:
+     - type: archive
+       url: https://deac-fra.dl.sourceforge.net/project/optipng/OptiPNG/optipng-7.9.1/optipng-7.9.1.tar.gz
+       sha256: c2579be58c2c66dae9d63154edcb3d427fef64cb00ec0aff079c9d156ec46f29
+
+  - name: comaps
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    cleanup:
+      - '/lib' # static libraries and build configs
+      - '/include'
+    sources:
+      - type: git
+        url: https://codeberg.org/comaps/comaps
+        tag: 2025.07.23-4-android
+        commit: 20c9fc5f45b37b62d58d45e1f179093c1b9a966a
+      - type: patch
+        path: ./desktopfile-icon.patch
+      - type: patch
+        path: ./disable-world-feed-integration-test.patch
+      - type: shell
+        commands:
+          - 'bash configure.sh'
+          - for s in {32,64,128,256,512}; do ffmpeg -i qt/res/logo.png
```

Comment:
logo.png is 512x512. Just install that one and leave the rest alone. There is no point to do that here.